Spec.md

```diff
@@ -198,31 +204,43 @@ status:
 
 The ExternalSecret control loop **ensures** that the target resource exists and stays up to date with the upstream provider. Because most upstream APIs are limited in throughput the control loop must implement some sort of jitter and retry/backoff mechanic.
 
-### External Secret Store
+### Secret Store
 
 The store configuration in an `ExternalSecret` may contain a lot of redundancy, this can be factored out into its own CRD.
-These stores are defined in a particular namespace using `SecretStore` **or** globally with `GlobalSecretStore`.
+These stores are defined in a particular namespace using `SecretStore` **or** globally with `ClusterSecretStore`.
 
 ```yaml
-apiVerson: external-secrets.k8s.io/v1alpha1
+apiVerson: external-secrets.x-k8s.io/v1alpha1
```

Comment:
AFAIK `x-k8s.io` is reserved for SIG sponsored projects. I'll try to find out what that actually means (couldn't find much in the k8s docs). This is not a blocker for this PR tho. We'll change that later i think

Reply:
Agreed. What do you think about only having one subdomain, `external-secrets.io`, like the examples below?

```yaml
apiVersion: external-secrets.io/v1alpha1
kind: ExternalSecret
metadata:
  name: externalsecret-sample
spec:
  # Add fields here
  foo: bar
```

```yaml
apiVersion: external-secrets.io/v1alpha1
kind: SecretStore
metadata:
  name: secretstore-sample
spec:
  # Add fields here
  foo: bar
```

Reply:
@knelasevero

Reply:
yeah, that's good!